src/components/Bookmark.js

```diff
@@ -1,19 +1,33 @@
 import '../css/Bookmark.css';
-import { useState } from 'react';
-import { FaBookmark } from 'react-icons/fa';
-import { FaRegBookmark } from 'react-icons/fa';
+import { useState, useContext } from 'react';
+import { FaBookmark as ActiveBookmark } from 'react-icons/fa';
+import { FaRegBookmark as InactiveBookmark } from 'react-icons/fa';
+import { UserSpellbookContext } from '../contexts/UserSpellbookContext';
 
-const Bookmark = () => {
-  const [isBookmarked, setIsBookmarked] = useState(false);
+const Bookmark = ({ spell }) => {
+  const [ isBookmarked, setIsBookmarked ] = useState(false);
```

Comment:
I might be missing something, but is `isBookmarked` being used in any capacity? It doesn't really make sense to be saving it in state or updating it if it isn't used to _do_ something.